packages/react-scripts/config/cordovaConfig.js

```diff
@@ -0,0 +1,53 @@
+// This file was written by Acorn developers
```

Comment:
Can we reference the Vue-cli file we copied this over from? The license on that package is MIT, so we don't run afoul of any license constraints by doing so

Reply:
Reference added.